test/Nancy.Testing.Tests/Nancy.Testing.Tests.csproj

```diff
@@ -26,12 +26,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170106-08" />
```

Comment:
Oh shit I thought I did that yesterday. I must've forgotten to push that commit.. If you check the `Changes performed in this pull-request` section above, I even added it after my last commit :D I'll push that out. Well spotted 👍